pages/guideforms.py

```diff
@@ -188,10 +198,9 @@ def validate(self, value):
         help_text=('The spec document has details in a specification language '
                    'such as Web IDL, or there is an exsting MDN page.')),
 
-    'spec_mentors': forms.EmailField(
+    'spec_mentors': MultiEmailField(
         required=False, label='Spec mentor',
-        widget=forms.EmailInput(
-            attrs={'multiple': True, 'placeholder': 'email'}),
+        widget=forms.EmailInput(attrs=MULTI_EMAIL_FIELD_ATTRS),
```

Comment:
Can this widget constructor argument be specified once in a new MultiEmailField constructor with a call to super() rather than each time?

Reply:
I'm not sure, but it sounds reasonable.   There is more in the python implementation of the super() that I didn't want to try to figure out just for this PR.